src/app/utils/formValidationUtils.js

```diff
@@ -156,19 +142,19 @@ function validate(form, cb) {
       validate: () => true,
       errorMsg: '',
     },
-    'starting-page': {
-      validate: (val) => isNumeric(val),
-      errorMsg: 'Please enter the starting page',
+    startPage: {
+      validate: (val) => (isNumeric(val) && val > '0'),
+      errorMsg: 'Please enter the start page',
     },
-    'ending-page': {
-      validate: (val) => isNumeric(val),
+    endPage: {
+      validate: (val) => (isNumeric(val) && val > '0'),
```

Comment:
Let's try to not use string comparisons like this, especially for numbers, because javascript isn't very good at doing so. Maybe you can explicitly convert it to a number and then check `val > 0`. In the background, javascript will try to convert the type for you and it can be prone to errors (but maybe es6/babel handles this better? Not too sure).